### PR TITLE
Refactored tests to make them work

### PR DESCRIPTION
**Pull request by tushartk**
1. Please verify the assertions in the `test_runner.py` scripts
2. I've also commented out `multi_env_performance` tests from `test_envs.py` for now, as they are now initialized in a different way. I will be working on those next.
3. Have also deleted `create_multi_env` function from `env_utils.py`. For now only `EnvCriticalError` Exception class remains, which could perhaps be shifted out to `doom_gym.py` as that's the only place where it is used.

### CODE REVIEW

File: envs/env_utils.py [hunk 1]
```diff
@@ -1,29 +1,2 @@
...
-    else:
-        from algorithms.utils.multi_env import MultiEnv
-        multi_env = MultiEnv(num_envs, num_workers, make_env_func, stats_episodes)
-
-    tmp_env.close()
-
-    return multi_env
-
-
 class EnvCriticalError(Exception):
```
Comment (alex-petrenko) [inline envs/env_utils.py:1]: I think you can leave it here, it's fine. Don't move it to Doom

File: envs/tests/test_envs.py [hunk 2]
```diff
@@ -55,36 +55,37 @@ def test_env_performance(make_env, env_type, verbose=False):
...
-                log.info('Collected %d frames of experience...', frames)
-                next_print += print_step
-
-    fps = total_num_frames / t.experience
-    log.debug('%s performance:', env_type)
-    log.debug('Took %.3f sec to collect %d frames in parallel, %.1f FPS', t.experience, total_num_frames, fps)
-    log.debug('Timing: %s', t)
-
-    multi_env.close()
+# def test_multi_env_performance(make_env, env_type, num_envs, num_workers, total_num_frames=1000):
```
Comment (alex-petrenko) [inline envs/tests/test_envs.py:61]: This test relied on the old class MultiEnv
Now we can rewrite it using DummySampler class
Or just remove this test completely

Comment (tushartk) [reply envs/tests/test_envs.py:61]: Sure, probably thought I would rewrite it

File: envs/env_utils.py (hunk 1 above)
Comment (tushartk) [reply envs/env_utils.py:1]: Perfect

**Pull request merged**